.github/actions/make-bundle/action.yml

```diff
@@ -41,25 +41,43 @@ inputs:
     description: "image uri for analyzer addon (ie. quay.io/<namespace>/<image-name>:<tag>)"
     required: false
     default: ""
+  version:
+    description: "operator version"
+    required: false
+    default: ""
+  channels:
+    description: "comma separated channel(s) this operator should be available on"
+    required: false
+    default: ""
+  # use_image_digest:
```

Comment:
are we leaving this in for a reason?

Reply:
For some strange reason when I use the `operator-sdk generate bundle ... --use-image-digests` on our project it doubles up the `relatedImages` where each image is repeated multiple times. I tried to recreate the issue on memcached-operator and couldn't. Not certain it's worth the time.

Dropping for now.